config/routes.rb

```diff
@@ -45,6 +45,9 @@
       # API V1 is in pre-release: Available iff api_v1 FeatureFlag is enabled
       constraints(->(_req) { FeatureFlag.enabled?(:api_v1) }) do
         scope module: :v1, constraints: ApiConstraints.new(version: 1, default: false) do
+          # V1 only endpoints
+          post "/articles/:id/unpublish", to: "articles#unpublish", as: :article_unpublish
```

Comment:
I named the route like this. If anyone has suggestions on alternatives they'd prefer I will definitely welcome them for open discussion 🙂